yarn-project/end-to-end/src/composed/integration_proof_verification.test.ts

```diff
@@ -0,0 +1,198 @@
+import { L2Block, deployL1Contract, fileURLToPath } from '@aztec/aztec.js';
+import { BBCircuitVerifier } from '@aztec/bb-prover';
+import { Fr, HEADER_LENGTH, Proof } from '@aztec/circuits.js';
+import { type L1ContractAddresses } from '@aztec/ethereum';
+import { type Logger } from '@aztec/foundation/log';
+import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
+import { AvailabilityOracleAbi, RollupAbi } from '@aztec/l1-artifacts';
+
+import { type Anvil } from '@viem/anvil';
+import { readFile } from 'fs/promises';
+import { join } from 'path';
+// @ts-expect-error solc-js doesn't publish its types https://github.com/ethereum/solc-js/issues/689
```

Comment:
Poor guy has even had a [PR posted for over a year](https://github.com/ethereum/solc-js/pull/693)